README.md

```diff
@@ -90,6 +91,161 @@ This WordPress project is intended to provide:
   define( 'SCRIPT_DEBUG', true );
   ```
 
+## Automated Testing
+
+This is a guide for setting up and using Codeception and WP-Browser. Codeception is an all-purpose testing framework for PHP.
+
+### Table of Contents ###
+
+1. [Introduction](#introduction)
+2. [Requirements](#requirements)
+3. [Installation](#installation)
+4. [Usage](#usage)
+5. [Directories](#directories)
+6. [Testing Suites](#testing-suites)
+7. [Database Dump](#database-dump)
+8. [Resources](#resources)
+
+### Introduction ###
+
+**Codeception**
+
+Codeception tests are written in the scenario driven manner and are very simple to write, read, and maintain. It can run acceptance, functional, unit tests, keeping all them inside one framework. As well, it is built on top of PHPUnit and is able to execute its tests.
+
+**Wp-Browser**
+
+WP-browser is a package built for using Codeception with WordPress. It uses a specific set of extensions to ease WordPress testing. This includes WordPress defined functions, constants, classes and methods in any test. See the [modules](https://github.com/lucatume/wp-browser#modules) for more specific information. 
+
+### Requirements
+
+* Composer (1.4.2)
+* Wp-Browser (1.21)
+* Codeception (2.3.4)
+
+
+_**note:** Wp-Browser will install the latest Codeception._
+
+### Installation
```

Comment:
Tests are essential part of an application so move these install steps into main Installation section above.